EDSAnalytics/EDSAnalytics.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
Typically we add a `Solution Items` folder to the solution file with common sample files like the README, pipeline, etc, see [here](https://github.com/osisoft/OSI-Samples-OCS/blob/master/basic_samples/Authentication/AuthorizationCodeFlow/DotNet/AuthorizationCodeFlow.sln) for an example.